packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -372,17 +372,28 @@ class FlutterPlugin implements Plugin<Project> {
     }
 
     private Properties getPluginList() {
-        File pluginsFile = new File(project.projectDir.parentFile.parentFile, '.flutter-plugins')
-        Properties allPlugins = readPropertiesIfExist(pluginsFile)
+
```

Comment:
nit: empty line - Typically we use empty lines to separate logical procedures for better readability.

Reply:
empty line removed